components/brave_vpn/resources/panel/components/select-region-list/style.ts

```diff
@@ -2,98 +2,161 @@
+
+import styled, { css } from 'styled-components'
+import Icon from '@brave/leo/react/icon'
+import Button from '@brave/leo/react/button'
 
 export const Box = styled.div`
-  --box-bg: ${(p) => p.theme.color.panelBackground};
   width: 100%;
   height: 100%;
-  background: var(--box-bg);
+  background: var(--leo-color-container-background);
   overflow: hidden;
 `
 
-export const PanelHeader = styled.section`
+export const PanelContent = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-itmes: flex-start;
+  gap: 0px;
+  align-self: stretch;
+`
+
+export const PanelHeader = styled.div`
   display: flex;
+  padding: var(--leo-spacing-2xl) var(--leo-spacing-2xl) var(--leo-spacing-xl)
+    var(--leo-spacing-2xl);
   align-items: center;
-  width: 100%;
-  padding: 18px 18px 0 18px;
-  margin-bottom: 18px;
-  box-sizing: border-box;
+  gap: var(--leo-spacing-m);
 `
 
-export const PanelContent = styled.section`
-  padding: 0 0 24px 0;
+export const Divider = styled.div`
+  height: 1px;
+  background: var(--leo-color-divider-subtle);
 `
 
 export const RegionList = styled.div`
-  padding: 0 18px;
-  max-height: 320px;
+  display: flex;
+  flex-direction: column;
+  padding: var(--leo-spacing-m) var(--leo-spacing-xl) var(--leo-spacing-m)
+    var(--leo-spacing-m);
+  align-items: center;
+  gap: var(--leo-spacing-m);
+  width: 320px;
+  max-height: 380px;
   overflow-y: scroll;
-
-  input {
-    /* on input click the browser focuses, which causes a jump
-    /* https://stackoverflow.com/a/49452792
-    */
-    top: auto;
-    left: -9999px;
-  }
+  // Due to vertical scrollbar's width, unnecessary horizontal scroll bar is
+  // shown.
+  overflow-x: hidden;
 
   &::-webkit-scrollbar {
     -webkit-appearance: none;
   }
 
   &::-webkit-scrollbar:vertical {
-    width: 8px;
+    width: 6px;
   }
 
   &::-webkit-scrollbar-thumb {
-    border-radius: 8px;
-    border: 2px solid var(--box-bg);
-    background-color: ${(p) => p.theme.color.divider01};
+    border-radius: 100px;
+    background-color: var(--leo-color-divider-subtle);
   }
+`
 
-  label {
-    align-items: center; // vertically aligns radio button with contents next to it
-  }
+export const RegionContainer = styled.div<{ selected: boolean }>`
+  width: 296px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  align-self: stretch;
+  border-radius: var(--leo-radius-m);
+  border: 1px solid var(--leo-color-divider-subtle);
+  background: var(--leo-color-container-background);
+
+  ${(p) =>
+    p.selected
+      ? css`
+          border: 1px solid var(--leo-color-divider-interactive);
+          background: var(--leo-color-container-interactive);
+          box-shadow: 0px var(--Elevation-xxs, 1px) 0px 0px
+              var(--Semantic-Elevation-Primary, rgba(0, 0, 0, 0.05)),
+            0px var(--Elevation-xxs, 1px) var(--Elevation-xs, 4px) 0px
+              var(--Semantic-Elevation-Secondary, rgba(0, 0, 0, 0.1));
```

Comment:
These tokens don't exist - I think you probably want

```suggestion
          box-shadow: ${effect.elevation['01']};
```